src/docs/releases/2.1.0.md

```diff
@@ -2,7 +2,19 @@
 
 Release date: Not yet released
 
-## Change Logs
+Here's the updated version to include the reference to `IEntity` implementations:
+
+## Change Log
+
+### Behavioral Changes
+
+#### `JsonOptions` Configuration
+
+A key change in version 2.0 is the shift from **Newtonsoft.Json** to **System.Text.Json**. Previously, we configured the default `JsonOptions` to match the settings used for document serialization. In this release, however, we’ve reverted that approach, and the `JsonOptions` are no longer configured by default. This change provides greater flexibility, allowing you to customize JSON serialization as needed.
+
+If your MinimalAPI returns YesSql documents, entities (implementations of `IEntity`) or content (implementations of `IContent`), such as `ContentItem`, `User`, `Query`, or `Notification`, you may need to use `DocumentJsonSerializerOptions` for proper serialization. For example, when using Minimal API, instead of returning the result with `TypedResults.Ok(entity)`, you should use `Results.Json(entity, options.Value.SerializerOptions)`, resolving `IOptions<DocumentJsonSerializerOptions>` from the IoC container.
```

Comment:
```suggestion
If your Minimal API returns YesSql documents, entities (implementations of `IEntity`) or content (implementations of `IContent`), such as `ContentItem`, `User`, `Query`, or `Notification`, you may need to use `DocumentJsonSerializerOptions` for proper serialization. For example, when using Minimal API, instead of returning the result with `TypedResults.Ok(entity)`, you should use `Results.Json(entity, options.Value.SerializerOptions)`, resolving `IOptions<DocumentJsonSerializerOptions>` from the IoC container.
```